sdk/keyvault/keyvault-keys/test/public/keyClient.spec.ts

```diff
@@ -490,35 +471,12 @@ describe("Keys client - create, read, update and delete operations", () => {
         });
       });
 
-      it("updateKeyRotationPolicy supports tracing", async () => {
-        const keyName = recorder.getUniqueName("updaterotationpolicy");
-        const key = await client.createKey(keyName, "EC");
-
-        await supportsTracing(
-          (tracingOptions) =>
-            client.updateKeyRotationPolicy(
-              key.name,
-              {
-                lifetimeActions: [
-                  {
-                    action: "Notify",
-                    timeBeforeExpiry: "P30D",
-                  },
-                ],
-                expiresIn: "P90D",
-              },
-              { tracingOptions }
-            ),
-          ["Azure.KeyVault.Keys.KeyClient.updateKeyRotationPolicy"]
-        );
-      });
-
       it("throws when attempting to fetch a policy of a non-existent key", async () => {
         const keyName = recorder.getUniqueName("nonexistentkey");
         await assert.isRejected(client.getKeyRotationPolicy(keyName));
       });
 
-      it("getKeyRotationPolicy supports tracing", async () => {
+      it.skip("getKeyRotationPolicy supports tracing", async () => {
```

Comment:
Why are you skipping this test?

Reply:
i had this qs too - a couple of supportsTracing tests are either skipped or only run in `live` mode. Is there a reason for this?

Reply:
Shoot, I forgot to re-enable it. Thanks!